todo/todo.py

```diff
@@ -183,344 +173,370 @@ async def showsettings(self, ctx):
 
     ### Listing commands ###
 
     @commands.group()
     async def todo(self, ctx):
-        """Base todo reminder command"""
+        """Base command for adding, removing, and completing todos"""
         pass
 
-    @todo.group(invoke_without_command=True)
-    async def complete(self, ctx, *indexes: positive_int):
-        """Complete todo reminders"""
-        # Thanks to Blackie#0001 on Red for the idea
-        # :D
-        if not len(indexes):
-            return await ctx.send_help(ctx.command)
-        indexes = self.sort_lists(items=indexes)
-        passes, fails = 0, 0
-        completed = []
-        async with ctx.typing():
-            async with self.config.user(ctx.author).todos() as todos:
-                for index in indexes:
-                    try:
-                        _ = todos.pop(index)
-                    except IndexError:
-                        fails += 1
-                    else:
-                        passes += 1
-                        completed.append(_)
-            # This stuff doesn't need to be in
-            # the `async with` part
-            msg = ""
-            detailed = await self.config.user(ctx.author).detailed_pop()
-            if passes:
-                if detailed:
-                    comp = [f"`{x}`" for x in completed]
-                    msg += f"Completed {passes} todos:\n{', '.join(comp)}\n"
-                else:
-                    msg += f"Completed {passes} todos\n"
-            if fails:
-                msg += f"Failed to complete {fails} todos"
-            if not msg:
-                msg = "Hm, something went wrong"
-        await self.maybe_reply(ctx, content=msg)
-        async with self.config.user(ctx.author).completed() as complete:
-            for item in completed:
-                complete.append(item)
-        # Call maybe sort here
-        await self._maybe_auto_sort(ctx.author)
+    @todo.command(name="add")
+    async def todo_add(self, ctx, *, todo: str):
+        """Add a todo to your list"""
+        async with self.config.user(ctx.author).todos() as todos:
+            todos.append(todo)
 
-    @complete.command(name="list")
-    async def complete_list(self, ctx):
-        """List your completed todos!"""
-        if len((completed := await self.config.user(ctx.author).completed())):
-            md = await self.config.user(ctx.author).use_md()
-            if not md:
-                completed = await self.cross_lists(completed)
-            completed = await self.number(completed)
-            await self.page_logic(
-                ctx,
-                completed,
-                "Completed Todo List",
-            )
-        else:
-            await ctx.send("You don't have any completed todos yet!")
+        msg = "Added that as a todo!"
+        details = await self.config.user(ctx.author).detailed_pop()
+        if details:
+            msg += f"\n'{discord.utils.escape_markdown(todo)}'"
+        await ctx.send(content=msg)
```

Comment:
No need to define content here (optional).

```suggestion
        await ctx.send(msg)
```